ecosystem/sep-0006.md

```diff
@@ -795,7 +789,8 @@ The response should be a JSON object like:
     }
   },
   "fee": {
-    "enabled": false
+    "enabled": false,
+    "description": "Fees vary based on the the assets transacted and method by which funds are delivered to or collected by the anchor."
```

Comment:
I would expect to see some more detailed information, such as a % for fee, or something. Do you think that is realistic how people will use the description? I worry by using a non-informative example here we will encourage similar non-information descriptions.

Reply:
I can make the description better here but I think some descriptions will inevitably be like this if they are sufficiently complex.

Reply:
I agree, that'll probably happen, but I think our example here should aim to contain the sorts of things we hope implementers will include.